src/lib.rs

```diff
@@ -829,6 +840,19 @@ impl Builder {
         self
     }
 
+    /// Set whether rustfmt should format the generated bindings.
+    pub fn format_bindings(mut self, doit: bool) -> Self {
+        self.options.format_bindings = doit;
+        self
+    }
+
+    /// Set the absolute path to the rustfmt configuration file, if None, the standard rustfmt
+    /// options are used.
+    pub fn format_configuration_file(mut self, path: Option<PathBuf>) -> Self {
+        self.options.format_configuration_file = path;
```

Comment:
This should probably imply `format_bindings` is `true`. That is, calling this method should also set `format_bindings` to `true.